glrm/glrm.py

```diff
@@ -60,17 +64,18 @@ def _initialize_probs(self):
         self.objY = 0
         Zx = self.Xv @ self.Yp
         Zy = self.Xp @ self.Yv
-        for columns,loss_fxn in self.loss_list:
-            for col in columns:
+        columns = self.A.shape[1]
+        for loss_fxn in self.loss_list:
+            for col in range(columns):
                 Acol = self.A[:,col][self.mask[:,col]]
                 Zxcol = Zx[:,col][self.mask[:,col]]
                 Zycol = Zy[:,col][self.mask[:,col]]
                 
 #                 Acol
 #                 print(col)
 #                 print((Acol,Zx[:,col]+self.mu[col].shape)
-                self.objX += loss_fxn(Acol,Zxcol+self.mu[col])/self.sigma[col]
-                self.objY += loss_fxn(Acol,Zycol+self.mu[col])/self.sigma[col]
+                self.objX += loss_fxn.loss(loss_fxn, Acol,Zxcol+self.mu[col])/self.sigma[col]
+                self.objY += loss_fxn.loss(loss_fxn, Acol,Zycol+self.mu[col])/self.sigma[col]
```

Comment:
Why these changes?

The loss objects implement `__call__` so you should be able ot just call `loss_fxn`.

https://github.com/ianhi/GLRM-1/blob/e1dfddfae0023b2f4b0a3d5abc850718752ab851/glrm/loss.py#L16